doc/whats_new.rst

```diff
@@ -499,6 +501,7 @@ People who contributed to this release  (in alphabetical order):
 * 	Marijn van Vliet
 * 	Mikolaj Magnuski
 * 	Peter J. Molfese
+*   Quentin Bertrand
```

Comment:
Remove, we haven't made the release notes for 0.18 yet (this list is from a previous release)